styles/dialog.css

```diff
@@ -1,5 +1,5 @@
 dialog {
-  display: flex;
+  display: none;
```

Comment:
i'll need to ask the others aswell, but for many prompts the display is set to flex[for now]... changing it here will change all hmm

Reply:
> i'll need to ask the others aswell, but for many prompts the display is set to flex[for now]... changing it here will change all hmm

wait never mind. discard that